.github/README.md

```diff
@@ -0,0 +1,24 @@
+# STLab CI Configuration
+
+The CI configuration in `workflows/build_and_test.yml` strives to be a minimal wrapper around the root-level `stlab.py` script, which will be invoked thrice (once each for `install`, `build`, and `test` subcommands) for each configuration listed in `.matrix.json`.
```

Comment:
I'd suggest separating the design "minimal wrapper around the script" from the details (invoked thrice). With the design you can write the "what" and the "why" together. Perhaps something like this.

> The CI configuration is designed as a minimal wrapper around `stlab-build-tool.py`. This enables reliable simulation of CI builds locally.

- redundant "script"
- root-level is obvious
- `stlab-build-tool.py` is more descriptive
- Tried to [omit needless words](https://en.wikisource.org/wiki/The_Elements_of_Style/Principles#13._Omit_needless_words.)